connector/saml/saml.go

```diff
@@ -411,6 +411,21 @@ func (p *provider) HandlePOST(s connector.Scopes, samlResponse, inResponseTo str
 	return ident, nil
 }
 
+// GetSAMLIssuer parses a SAML response and returns its 'Issuer' field.
+func GetSAMLIssuer(samlResponse string) (string, error) {
+	rawResp, err := base64.StdEncoding.DecodeString(samlResponse)
+	if err != nil {
+		return "", err
+	}
+
+	var resp response
+	if err := xml.Unmarshal(rawResp, &resp); err != nil {
+		return "", err
+	}
+
+	return resp.Issuer.Issuer, nil
```

Comment:
❓ Could `resp.Issuer` be `nil`? (I really don't know! 😅)

Reply:
good call, probably doesn't hurt to check